nnvm/tutorials/tune_nnvm_arm.py

```diff
@@ -307,7 +307,8 @@ def tune_tasks(tasks,
                 tuner_obj.load_history(autotvm.record.load_from_file(tmp_log_file))
 
         # do tuning
-        tuner_obj.tune(n_trial=min(n_trial, len(tsk.config_space)),
+        n_trial=min(n_trial, len(tsk.config_space))
```

Comment:
nit: please change formatting to
`n_trial = min(n_trial, len(tsk.config_space))`
(add space around `=`)